components/MashCards/MainCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { MashReferral } from 'types';
+import ContactCard from './ContactCard';
+import InitialDecisionCard from './InitialDecisionCard';
+import ScreeningCard from './ScreeningCard';
+import FinalDecisionCard from './FinalDecisionCard';
+
+interface Props {
+  filter: string;
+  mashReferrals: MashReferral | any;
+}
+
+const MainCard = ({ filter, mashReferrals }: Props): React.ReactElement => {
+  if (filter === 'contact') {
+    return (
+      <div>
+        {mashReferrals.map((referral: any) => (
```

Comment:
for all of these maps we will get rid of the :any once the any is removed from the incoming prop.

Typescript should then correctly infer that the referral is of type MashReferral

Reply:
For some reason when I remove 'any' on the second map then error messages arise so I need to investigate this further